app/packages/core/src/components/Filters/NumericFieldFilter.tsx

```diff
@@ -283,7 +283,7 @@ const NumericFieldFilter = ({
             formatter={
               [DATE_TIME_FIELD, DATE_FIELD].includes(ftype)
                 ? (v) => (v ? formatDateTime(v, timeZone) : null)
-                : (v) => null
+                : (v) => (v ? v.toString() : null)
```

Comment:
This should do a `typeof` check so `<= 0` values are also shown
```suggestion
                : (v) => (typeof v !== "number" ? v.toString() : null)
```

Reply:
@benjaminpkane Thanks for the catch!!!